scripts/eurostat/health_determinants/common/download.py

```diff
@@ -49,7 +49,7 @@ def download_files(download_files_url: list, download_path: str) -> None:
 
     Args:
         download_file_url (str): url of the file to be downloaded as a string
-        download_path (str): local directory to dlownload the file
+        download_path (str): local directory to download the file
```

Comment:
Please add a try-catch block to gracefully handle potential download failures. If an error occurs, log the error using 'logging.fatal' and terminate the script.

Reply:
Addressed the comment, modified the code accordingly.